monitor/grafana/zeebe.json

```diff
@@ -11185,6 +11188,583 @@
       ],
       "title": "Start up",
       "type": "row"
+    },
+    {
+      "collapsed": false,
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 14
+      },
+      "id": 315,
+      "panels": [],
+      "title": "Backups",
+      "type": "row"
+    },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "custom": {},
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": []
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 15
+      },
+      "id": 329,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "text": {},
+        "textMode": "auto"
+      },
+      "pluginVersion": "7.4.5",
+      "targets": [
+        {
+          "exemplar": false,
+          "expr": "min(max_over_time(zeebe_backup_operations_in_progress{namespace=~\"$namespace\", partition=~\"$partition\", pod=~\"$pod\", operation=\"take\"}[5m])) by (partition)",
```

Comment:
:question: This query is a bit weird. I tried to remove time series where a gauge would be stuck at a specific value. This happened during testing when a broker would transition to follower while taking a backup. Because that follower never saw the backup complete or fail it would still report an in-progress backup. 

I've tried to explain the query I came up here but ultimately I'm not sure why exactly this works or even if it is correct at all :see_no_evil:

Reply:
May be we should also update the in progress metrics while closing BackupService. If the node restarts, the metrics is reset anyway, I think, as observed in other metrics. Can you verify it? If it works, then we probably don't need this complex query.

In this query, it is not clear what would be the value if inprogress backup takes more than 5m. Will max_over_time query ignored it and shows the result as 0?

Reply:
> May be we should also update the in progress metrics while closing BackupService

Good idea. Alternatively we could update when marking the in-progress backup as failed during startup of a new BackupService.

Reply:
I fixed it by resetting the in-progress counters to 0 when the BackupService actor closes. Query is adjusted too.